scenes/enumerated/problem_12629/text.py

```diff
@@ -0,0 +1,6 @@
+from constants import LanguageConfig
+
```

Comment:
Define a variable `__all__` which is a list of variable names (as string), so that `import *` will import only those. No need to define in `armenian_text` and `english_text`, as there are no other variables (e.g. import * from here will also import LanguageConfig, but if you don't specify it in `__all__`, it will not be imported)

Reply:
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package